apps/server/src/server.js

```diff
@@ -84,8 +84,9 @@ app.get("*", (req, res) => {
   res.sendStatus(404);
 });
 
-app.listen(config.PORT, () => {
-  const message = `Server is running on port ${config.PORT}, ENV: ${config.ENV}, LOG_LEVEL: ${config.LOG_LEVEL}`;
+const port = process.env.PORT || config.PORT
```

Comment:
Why do we need this line? config.PORT is getting it's value from process.env.PORT

Reply:
Oh yeah that's true this is not necessary